ledger/participant-integration-api/src/main/resources/db/migration/postgres/V45__fix_large_size_index_issues.sql

```diff
@@ -0,0 +1,5 @@
+-- Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+DROP INDEX IF EXISTS participant_command_completio_completion_offset_application_idx;
```

Comment:
Why the `IF EXISTS` clause? The list of previous migrations should determine whether the index exists or not.

Reply:
Fixed.